runtime/polkadot/src/voter_bags.rs

```diff
@@ -0,0 +1,234 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
```

Comment:
file to be removed.

Reply:
3b4b02b1d0867a1525867828ae20d5a4ab74400b